### PR TITLE
refactor(sdk): change scss files names, add common files for variable…

### PR DESCRIPTION
**Pull request by papitaanda**
…s and mixins for sass, use mixins and variables in other scss files

### CODE REVIEW

File: src/base/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@mixin size($width, $height) {
+  width: $width;
+  height: $height;
+}
+
+@mixin padding($top, $bottom, $left, $right) {
```
Comment (bogdansimon) [inline src/base/_mixins.scss:6]: can we mark all these as optional somehow? And check against null, similarly to the flexContainer mixin

File: src/controls/GridControl/_grid.scss [hunk 2]
```diff
@@ -1,3 +1,13 @@
+%modal-button {
+  border-radius: 32px;
+  padding: 16px;
```
Comment (bogdansimon) [inline src/controls/GridControl/_grid.scss:3]: can these padding props be replaced with the 'padding' mixin? After marking its arguments as optional

File: src/controls/GridControl/_grid.scss [hunk 3]
```diff
@@ -18,16 +28,14 @@
 
 .mapbox-control-grid-selector-modal {
   &-root {
-    width: 100%;
-    height: 100%;
+    @include size(100%, 100%);
     display: none;
     position: fixed;
     z-index: 1;
     left: 0;
```
Comment (bogdansimon) [inline src/controls/GridControl/_grid.scss:35]: Perhaps we can create an 'absolutePosition' mixin too, to specify the top, left, right and bottom rules

File: src/controls/GridControl/_grid.scss [hunk 4]
```diff
@@ -42,17 +50,14 @@
     border-radius: 16px;
   }
   &-header {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
+    @include flexContainer(null, center, space-between);
     padding: 6px 30px;
     background-color: #fff;
```
Comment (bogdansimon) [inline src/controls/GridControl/_grid.scss:55]: we can define 'White' and 'Black' in the variables file too

File: src/controls/GridControl/_grid.scss (hunk 3 above)
Comment (papitaanda) [reply src/controls/GridControl/_grid.scss:35]: i'm not sure what rules to add, theres just one case where we have rules but it's according to the button placement (mapTilesControlPosition)

**Pull request merged**